tests/test_basic.py

```diff
@@ -1222,6 +1222,26 @@ def index():
         t.join()
 
 
+def test_werkzeug_passthrough_errors(monkeypatch):
+    rv = {}
+
+    # Mocks werkzeug.serving.run_simple method
+    def run_simple_mock(*args, **kwargs):
+        rv['passthrough_errors'] = kwargs.get('passthrough_errors')
+
+    app = flask.Flask(__name__)
+    monkeypatch.setattr(werkzeug.serving, 'run_simple', run_simple_mock)
+    for debug in [True, False]:
```

Comment:
Yep, makes sense. I'm not super familiar with py.test, I wasn't aware you can do that. I'll change it.